### PR TITLE
Feature vpc module

### PR DESCRIPTION
**Pull request by lowkent2me**
VPC module completed

### CODE REVIEW

File: .idea/.gitignore
Comment (ggramal) [inline .idea/.gitignore:1]: lets put `.idea/*` to .gitignore

Comment (ggramal) [reply .idea/.gitignore:1]: and remove those files ofcourse

Comment (lowkent2me) [reply .idea/.gitignore:1]: done

**Review by ggramal — APPROVED** [review]
Lets go

**Pull request merged**